recipe/meta.yaml

```diff
@@ -22,18 +29,44 @@ build:
   skip: True  # [osx and x86_64 and (py == 38)]
   # TODO: support Linux on ppc64le: https://github.com/catboost/catboost/issues/2145
   skip: True  # [ppc64le]
+  # Wheels from PyPI have a single version that already contains CUDA support so do not do separate builds on Windows,
+  # just have a single version for now
+  # TODO: build from source on Windows
+  skip: True  # [win and cuda_compiler_version != "None"]
+  # TODO: support CUDA >= 12.3 : https://github.com/catboost/catboost/issues/2755
+  skip: True  # [cuda_compiler_version not in ("None", "11.8", "12.0", "12.1", "12.2")]
```

Comment:
This is unfortunate, because it drops it effectively drops support for CUDA 12.x builds here. Based on your comments in https://github.com/catboost/catboost/issues/2755, I would expect build failures, which weren't present with our 12.6 setup.

But assuming we really rule out building with 12.6, then we should at least build for 12.0. This takes a bit of extra-effort because basically no other feedstock needs 12.0 anymore, but it can be done by overriding the respective zip in `recipe/conda_build_config.yaml`, [like so](https://github.com/conda-forge/pytorch-cpu-feedstock/blob/ceef60c1b6e1636a52edecfd716ef4520d62caa4/recipe/conda_build_config.yaml#L14-L39)

Reply:
> which weren't present with our 12.6 setup.

They were: https://github.com/conda-forge/catboost-feedstock/runs/36314493194, that's why I added this patch.

>  we should at least build for 12.0.
> it can be done by overriding the respective zip in recipe/conda_build_config.yaml, [like so](https://github.com/conda-forge/pytorch-cpu-feedstock/blob/ceef60c1b6e1636a52edecfd716ef4520d62caa4/recipe/conda_build_config.yaml#L14-L39)

Ok, I'll try this approach then.